src/command/restore/restore.c

```diff
@@ -313,18 +311,9 @@ restoreBackupSet(void)
                         }
                     }
 
-                    // If a backup was found on this repo matching the criteria for time then exit, else determine if the latest
-                    // backup from this repo might be used
+                    // If a backup was found on this repo matching the criteria for time then exit
                     if (found)
                         break;
-                    else
-                    {
-                        // If a backup was not yet found then set the latest from this repo as the backup that might be used
-                        RestoreBackupData candidate = restoreBackupData(
-                            latestBackup.backupLabel, repoIdx, infoPgCipherPass(infoBackupPg(infoBackup)));
-
-                        lstAdd(backupCandidateList, &candidate);
-                    }
                 }
                 else
                 {
```

Comment:
line 320 comment, should "(or time provided was not valid)" be removed.  Would the error thrown in getEpoch prevent us from reaching this point due to an invalid date?

Reply:
Fixed at f6c1549b6b.